config/features.yml

```diff
@@ -32,6 +36,10 @@ features:
     actor_type: user
     description: >
       https://www.va.gov/profile/ show Receive Text Notifications
+  profile_show_profile_2.0:
```

Comment:
and this is for this ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/6269